solutions/observability/apm/resource-attributes.md

```diff
@@ -36,13 +40,119 @@ processors:
 ...
 ```
 
-::::{tip}
-Need to add event attributes instead? Use attributes—​not to be confused with resource attributes—​to add data to span, log, or metric events. Attributes can be added as a part of the OpenTelemetry instrumentation process or with the [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor).
+## Handling of unmapped attributes
```

Comment:
This section is under resource attributes, but these rules apply to scope and record level attributes as well. Some applies to span level only.

Reply:
Could we solve this by moving the APM transactions and spans, as well as the Conditional attribute translation to spans docs?